serverlessworkflow/modules/ROOT/pages/cloud/operator/global-configuration.adoc

```diff
@@ -0,0 +1,95 @@
+= Global Configuration Settings
+:compat-mode!:
+// Metadata:
+:description: Global Configuration {operator_name} for cluster admins
+:keywords: sonataflow, workflow, serverless, operator, kubernetes, minikube, openshift, containers, configuration
+// links
+
+This document describes how to set global configuration options for the {operator_name}.
+
+.Prerequisites
+* You have installed the operator in the target cluster. You can find more information at the xref:cloud/operator/install-serverless-operator.adoc[] guide.
+
+== Modifying configuration options
+
+After installing the operator, you can access the ConfigMap named `sonataflow-operator-controllers-config` in the namespace `{operator_installation_namespace}`.
```

Comment:
`sonataflow-operator-controllers-config` 

could we make this a variable? Thinking if we ever update the name, we then just change one variable in docs.